src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -260,6 +348,26 @@ public void addExternalTrigger(ExternalTrigger external_trigger) {
 		return BlocklyVariables.isPlayerVariableForWorkspace(mcreator.getWorkspace(), field);
 	}
 
+	/**
+	 * Gets the readable name of a data list entry from the type of searchable selector
+	 *
+	 * @param value The value of the data list entry
+	 * @param type The type of the searchable selector
+	 * @return The readable name of the passed entry, or an empty string if it can't find a readable name
+	 */
+	@SuppressWarnings("unused") public String getReadableNameOf(String value, String type) {
+		String datalist;
+		switch (type) {
+			case "entity" -> datalist = "entities";
+			case "biome" -> datalist = "biomes";
+			default -> {
+				return "";
+			}

```

Comment:
Does IDEA require braces here or what?

Reply:
It gives me a compilation error without them, I guess that's a limitation of the enhanced switch

Reply:
Weird, but OKay